substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVSymbolSubsectionBuilder.java

```diff
@@ -55,9 +53,8 @@
      * The CodeView symbol section Prolog is also a CVSymbolSubsection, but it is not build in this
      * class.
      */
-    void build(DebugContext theDebugContext) {
-        this.debugContext = theDebugContext;
-        this.lineRecordBuilder = new CVLineRecordBuilder(debugContext, cvDebugInfo);
+    void build() {
+        this.lineRecordBuilder = new CVLineRecordBuilder(cvDebugInfo);
```

Comment:
Initialization could now be moved to the constructor.

Reply:
Done.